backend/scripts/get_balance.cdc

```diff
@@ -0,0 +1,13 @@
+import FungibleToken from 0xee82856bf20e2aa6
```

Comment:
I think it would be clearer if all these values were placeholders to get replaced. token contract name, address & public capability path